elfpy/data/postgres.py

```diff
@@ -0,0 +1,106 @@
+"""Initialize Postgres Server"""
+
+from sqlalchemy import Column, ForeignKey, Integer, String, create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+from elfpy.data.pool_info import PoolInfo
+
+Base = declarative_base()
+
+# replace the user, password, and db_name with credentials
+engine = create_engine("postgresql://admin:password@localhost:5432/postgres_db")
+
+
+class UserTable(Base):
+    """User Schema"""
+
+    __tablename__ = "users"
+
+    # address
+    id = Column(String, primary_key=True)
+
+
+class TransactionTable(Base):
+    """Transactions Schema"""
+
+    __tablename__ = "transactions"
+
+    id = Column(String, primary_key=True)
+    user_id = Column(String, ForeignKey("users.id"))
+    amount = Column(Integer)
+
+
+class PoolInfoTable(Base):
```

Comment:
I wonder if there's a way to automatically get the schema between the data class and this, so that the schema lives in one place (we'll have to do checks to ensure the current data class adheres to the existing data schema on postgres)

Reply:
I love this and yes I think we can accomplish this with some codegen tools!